.github/workflows/ci.yml

```diff
@@ -38,14 +38,3 @@ jobs:
       env:
         TOXENV: ${{ matrix.toxenv }}
       run: tox
-    - name: Run check-keywords
-      # This will analyze django apps to see if any of the model fields
-      # have names that are on a list of reseverd keywords and should not
-      # be used.
-      run: |
-        pip install -r requirements/base.txt
-        cookiecutter --no-input cookiecutter-django-ida
```

Comment:
This check was running just for `cookiecutter-django-ida`, but this commit moves it to the `python-template`, which is the base for all of the cookiecutters -- so now it's run for libraries, django apps, and xblocks (which don't have the make target). I assume this was unintentional, so I'm going to see if I can install this file just for the IDA cookiecutter.